packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -368,6 +491,42 @@ class _ZoomExitTransition extends StatelessWidget {
   }
 }
 
+class _ZoomExitTransitionDelegate extends AnimatedRasterDelegate {
+  const _ZoomExitTransitionDelegate({required this.reverse});
+
+  final bool reverse;
+
+  @override
+  int get hashCode => reverse.hashCode;
+
+  @override
+  bool operator ==(Object other) => other is _ZoomExitTransitionDelegate && other.reverse == reverse;
+
+  @override
+  bool willPaint(Animation<double> animation) {
+    final double fade = reverse
+      ? _ZoomExitTransition._fadeOutTransition.evaluate(animation)
+      : 1.0;
+    return fade != 0.0;
+  }
+
+  @override
+  void paint(PaintingContext context, ui.Image image, double pixelRatio, Animation<double> animation) {
+    final double fade = reverse
+      ? _ZoomExitTransition._fadeOutTransition.evaluate(animation)
+      : 1.0;
+    final double scale = (reverse
```

Comment:
Can scale end up being larger than 1.0? Will this make the scaled image more pixelated?